docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -461,12 +458,39 @@ is external so it's handy for Getting Started to share -->
         <title>Auto Scale core concepts</title>
         <para>To use the &PROD; API effectively, you should understand
             several key concepts: </para>
-        <section xml:id="Glossary">
-            <title>Glossary</title>
-            <!-- Glossary section, defining basic terms,
-is external so it's handy for Getting Started to share -->
-            <xi:include href="chapters/glossary.xml" xpointer="element(Glossary-000)"/>
-            <!-- -->
+        <section xml:id="How_Auto_Scale_Works">
+            <title>How Auto Scale Works</title>
+            <para>Rackspace &PROD; is an API-based tool that automatically scales resources in response to
+                an increase or decrease in overall workload based on user-defined thresholds.</para>
+            <para>&PROD; functions by linking these services:
+                <itemizedlist>
+                    <listitem>
+                        <para>&PROD; API, described in this guide.</para>
```

Comment:
I'm not sure what this looks like when it's built, but it sounds like if `&PROD;` is "Auto Scale", then it'd say something like:

Auto Scale functions by linking these services: 
- Auto Scale API, described in this guide, 
- Load-balanced Cloud Servers.  <link to load balancers>

Should the first bullet point be "Rackspace Cloud Servers" and the second one be "Cloud Load Balancer service", since that's the link?